pkg/justification/processor.go

```diff
@@ -98,7 +102,14 @@ func (p *Processor) CreateToken(ctx context.Context, request *jvspb.CreateJustif
 }
 
 func (p *Processor) getLatestSigner(ctx context.Context) (*signerWithID, error) {
-	ver, err := jvscrypto.GetLatestKeyVersion(ctx, p.kms, p.config.KeyName)
```

Comment:
Can we update the comment for `getLatestSigner` to note that it's an expensive operation that communicates with the upstream database?